onnx_tf/gen_status.py

```diff
@@ -20,67 +20,46 @@ def main(docs_dir):
   base_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
   docs_dir = os.path.join(base_dir, 'doc')
   onnx_version = ''
-  onnx_path = ''
   onnx_tf_release_build = False
 
   try:
-    opts, args = getopt.getopt(
-        sys.argv[1:], 'hv:p:r',
-        ['onnx_version=', 'onnx_path=', 'onnx_tf_release_build'])
+    opts, args = getopt.getopt(sys.argv[1:], 'hv:r',
+                               ['onnx_version=', 'onnx_tf_release_build'])
   except getopt.GetoptError:
-    print('gen_status.py -v <onnx_version> -p <onnx_path> ' +
-          '-r <onnx_tf_release_build>')
-    print('If onnx_version=master then onnx_path is required.')
+    print('gen_status.py -v <onnx_version> -r <onnx_tf_release_build>')
```

Comment:
Is -r <onnx_tf_release_build> accurate? Seems the code does not use <onnx_tf_release_build>